entries/SWC-133.md

```diff
@@ -0,0 +1,19 @@
+# Title
```

Comment:
Please rename the file to SWC-126 per #215.

Reply:
Fixed in https://github.com/SmartContractSecurity/SWC-registry/pull/223